profile-widget/diveprofileitem.cpp

```diff
@@ -46,17 +46,17 @@ void AbstractProfilePolygonItem::clipStop(double &x, double &y, double prev_x, d
 
 std::pair<double, double> AbstractProfilePolygonItem::getPoint(int i) const
 {
-	double x = dataModel.index(i, hDataColumn).data().toReal();
```

Comment:
I'd love to go back in time to figure out why we ever thought that was a good idea... were we thinking to have a different x axis? Plotting time over depth? Or depth of over partial pressures or...?
I have no idea